### PR TITLE
Hawkular Services 0.0.2.Final blog post

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
Ready for review.

File: src/main/jbake/content/blog/2016/06/14/hawkular-services-0.0.2.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, release, hawkular-services
+
+Hawkular Services is a new base line distribution including only the basic Hawkular components, such as Alerts, Inventory and Metrics.
+The version https://repository.jboss.org/nexus/content/groups/public/org/hawkular/services/hawkular-services-dist/0.0.2.Final/hawkular-services-dist-0.0.2.Final.zip[0.0.2.Final] has just been released and is ready to be tested and integrated with other projects.
+
+Notable differences between the main Hawkular distribution and Hawkular Services are:
+
+* Hawkular Services does not ship with an User Interface. Interaction with the backend is exclusively done via REST.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/06/14/hawkular-services-0.0.2.Final.adoc]: a User Interface

**Comment by pilhuhn** [comment]
You could link to http://www.hawkular.org/blog/2016/04/28/new-packaging.html and mention that h-c-s is now h-services

**Comment by pilhuhn** [comment]
Thanks @jpkrohling 👍

**Pull request merged**